build.sbt

```diff
@@ -148,7 +149,21 @@ lazy val site = project
   )
   .dependsOn(kernel, githubActions, noPublish)
 
+lazy val unidoc = project
+  .in(file("unidoc"))
+  .enablePlugins(TypelevelUnidocPlugin)
+  .settings(
+    // Doesn't conflict w/ core artifact since this one won't have the sbt cross version suffix
+    // javadoc.io doesn't recognize sbt artifacts, so we must publish like this
+    name := "sbt-typelevel",
```

Comment:
In retrospect not sure if naming the artifact `sbt-typelevel` is a good idea. `sbt-typelevel-docs` or `sbt-typelevel-unidoc` etc. is much better.